local HitRegister = require(game.ServerStorage.Modules.HitService)
local PlayerService = require(game.ServerStorage.Modules.PlayerService)
local ThrowService = require(game.ServerStorage.Modules.ThrowService)
local BrainrotService = require(game.ServerStorage.Modules.ActiveBrainrotService)
local NPCService = require(game.ServerStorage.Modules.NPCService)
local CurrencyService = require(game.ServerStorage.Modules.CurrencyService)
local BrainService = require(game.ServerStorage.Modules.BrainService)

HitRegister.Initialize()
ThrowService.Initialize()
-- NPCService.Initialize()
BrainService.Initialize()

game.Players.PlayerAdded:Connect(function(player)
	-- PlayerService.SetupPlayer(player)
	CurrencyService.Initialize(player)
	PlayerService.Initialize(player)
	player.CharacterAdded:Connect(function(character)
		BrainrotService.Initialize(player)
	end)
end)
